docs/indepth/installation.rst

```diff
@@ -173,49 +175,49 @@ Using Amazon Web Services
 
 #. Add the *Key* to your local key chain (download it from the AWS web interface first if necessary):
 
-   .. code-block:: sh
+   .. prompt:: bash laptop$
```

Comment:
This is better than it was, but annoyingly exposes us more to https://clusterhq.atlassian.net/browse/FLOC-1548 and we still don't have consistent prompts everywhere. I'll leave the judgement of which prompt to use, but I think that for now it should be the most common one. I think that might be `alice@mercury`. A follow-up can be made to improve them all.